keel-web/src/test/kotlin/com/netflix/spinnaker/keel/apidocs/ApiDocCompatibilityTests.kt

```diff
@@ -36,7 +35,7 @@ import strikt.api.expectThat
   webEnvironment = NONE
 )
 @AutoConfigureMockMvc
-@EnableAutoConfiguration(exclude = [TaskSchedulingAutoConfiguration::class])
+@DisableSpringScheduling
```

Comment:
Most of the remaining changes are just swapping the longer form out for the convenient new annotation